packages/pg-pubsub/src/PgGenericSubscriptionPlugin.ts

```diff
@@ -142,6 +174,12 @@ const PgGenericSubscriptionPlugin: Plugin = function (
               topic: {
                 type: new GraphQLNonNull(GraphQLString),
               },
+              immediate: {
+                defaultValue: false,
+                type: GraphQLBoolean,
```

Comment:
Let's turn `immediate: Boolean = false` to `immediate: Boolean! = false`.

```suggestion
                type: new GraphQLNonNull(GraphQLBoolean),
```